mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -76,10 +76,15 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 		ARG 0 hue
 		ARG 1 saturation
 		ARG 2 value
-	METHOD method_15370 minusDiv (DDD)D
-		ARG 0 numerator
-		ARG 2 delta
-		ARG 4 denominator
+	METHOD method_15370 inverseLerp (DDD)D
+		COMMENT Essentially does the inverse of the linear interpolation (lerp) function.
```

Comment:
Maybe say given delta, a, b, `inverseLerp(lerp(delta, a, b), a, b) == delta`? I thought it's like lerping the inversed `value` at first sight.